akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -73,7 +73,7 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
     case event @ Warning(logSource, logClass, message) ⇒
       withMdc(logSource, event) {
         event match {
-          case e: LogEventWithCause ⇒ Logger(logClass, logSource).warn(markerIfPresent(event), if (message != null) message.toString else null, e.cause)
+          case e: LogEventWithCause ⇒ Logger(logClass, logSource).warn(markerIfPresent(event), if (message != null) message.toString else e.cause.getLocalizedMessage, e.cause)
```

Comment:
but why localized, we usually do getMessage

Reply:
it's done in the `Error`-case as well, could change it of course.

Reply:
Hmm, I see... consistency wins -- let's keep what you did here :)